assets/timeTableApiHandler.js

```diff
@@ -24,9 +24,9 @@ export default class TimeTableApiHandler {
     } else {
       ticketPromise = this.getAllTickets().then((data) => {
         const result = data.result;
-        let tickets = result.filter(
-          (result) => result.type.toLowerCase() === "task",
-        );
+          let tickets = result.filter(
+              ({type}) => type.toLowerCase() !== "story" && type.toLowerCase() !== "milestone"
```

Comment:
Please add prettier to the pr.yml
it would look like this: 
https://github.com/ITK-Leantime/leantime-dataexport/blob/main/.github/workflows/pr.yml